### PR TITLE
Http Piece Retrival Test

### PR DESCRIPTION
**Pull request by stephen-pl**
#37 in RetrievalBot tasks

### CODE REVIEW

File: integration/spadev0/util.go [hunk 1]
```diff
@@ -164,7 +191,7 @@ func addErrorResults(
 		Task: task.Task{
 			Requester: requester,
 			Module:    "spadev0",
-			Metadata:  spadev0Metadata,
+			Metadata:  make(map[string]string),
```
Comment (stephen-pl) [inline integration/spadev0/util.go]: Wasnt sure which metadata to use for the error unless we want to pass it around

File: pkg/env/env.go [hunk 2]
```diff
@@ -2,47 +2,49 @@ package env
...
-	ResultMongoDatabase           Key = "RESULT_MONGO_DATABASE"
-	FilplusIntegrationBatchSize   Key = "FILPLUS_INTEGRATION_BATCH_SIZE"
-	FilplusIntegrationTaskTimeout Key = "FILPLUS_INTEGRATION_TASK_TIMEOUT"
-	FilplusIntegrationRandConst   Key = "FILPLUS_INTEGRATION_RANDOM_CONSTANT"
-	StatemarketdealsMongoURI      Key = "STATEMARKETDEALS_MONGO_URI"
-	StatemarketdealsMongoDatabase Key = "STATEMARKETDEALS_MONGO_DATABASE"
-	StatemarketdealsBatchSize     Key = "STATEMARKETDEALS_BATCH_SIZE"
-	StatemarketdealsInterval      Key = "STATEMARKETDEALS_INTERVAL"
-	PublicIP                      Key = "_PUBLIC_IP"
+	AcceptedContinents            Key = "ACCEPTED_CONTINENTS"
```
Comment (stephen-pl) [inline pkg/env/env.go:16]: Sorted this and got rid of the fil+ and statemarket deals

File: integration/spadev0/util.go (hunk 1 above)
Comment (jcace) [reply integration/spadev0/util.go]: hmm, have a look at the`addErrorResults` func in `integration/filplus/util.go` to see how it's done there. I originally copied from here but simplified as we only had one type - might be better to copy what they have.

**Review by jcace — APPROVED** [review]
LGTM!

**Pull request merged**